sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_client.py

```diff
@@ -613,12 +670,45 @@ def import_key(self, name, key, **kwargs):
         )
         return KeyVaultKey._from_key_bundle(bundle)
 
+    @distributed_trace
+    def release_key(self, name, target, version=None, **kwargs):
+        # type: (str, str, Optional[str], **Any) -> KeyReleaseResult
+        """Releases a key.
+
+        The release key operation is applicable to all key types. The target key must be marked
+        exportable. This operation requires the keys/release permission.
+
+        :param str name: The name of the key to get.
+        :param str target: The attestation assertion for the target of the key release.
+        :param str version: (optional) A specific version of the key to release. If unspecified, the latest version is
+            released.
+
+        :keyword algorithm: The encryption algorithm to use to protect the released key material.
+        :paramtype algorithm: ~azure.keyvault.keys.KeyExportEncryptionAlgorithm
+        :keyword str nonce: A client-provided nonce for freshness.
+
+        :return: The result of the key release.
+        :rtype: ~azure.keyvault.keys.KeyReleaseResult
+        :raises: :class:`~azure.core.exceptions.HttpResponseError`
+        """
+        result = self._client.release(
+            vault_base_url=self._vault_url,
+            key_name=name,
+            key_version=version or "",
```

Comment:
`None` isn't functionally the same?

Reply:
Passing `None` instead of a string raises an error, presumably because `key_version` is technically marked as required in the swagger (even though the service doesn't treat it as required). `get_key` and `update_key_properties` use this same `version or ""` pattern for, I assume, a similar reason